src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstLag;
+    private int secondLag;
```

Comment:
The variable names `firstLag` and `secondLag` might be confusing. Consider renaming them to `width` and `height` or `length` and `breadth` for better clarity and understanding.